core/buffer.lua

```diff
@@ -357,11 +389,14 @@ text.
 function reduxbuffer:add_text(text, style, command, indicator)
   text = tostring(text)
   local insert_pos = self.target.current_pos
+  local saved_buf = buffer
+  _G.buffer = self.target
```

Comment:
I see, e.g. a style that is applied. 

Could you add a general comment (just to help remember why this was done later).